operate/client/src/modules/hooks/useProcessInstancesFilters.test.ts

```diff
@@ -88,7 +85,7 @@ describe('useProcessInstanceFilters', () => {
         incidentErrorHashCode: 321456,
         hasRetriesLeft: true,
       },
-    };
+    } as const;
```

Comment:
🔧 The const casting has no effect, right? The type of `expectedRequest` will still be `GetProcessDefinitionStatisticsRequestBody` since it's explicitly typed.